plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -66,8 +80,14 @@ jQuery( function( $ ) {
 				$container 	= $( this ).closest( '.stars' );
 
 			$rating.val( $star.text() );
-			$star.siblings( 'a' ).removeClass( 'active' );
-			$star.addClass( 'active' );
+			$star.siblings( 'a' )
+				.removeClass( 'active' )
+				.attr( 'aria-checked', 'false' )
+				.attr( 'tabindex', '-1' );
+			$star
+				.addClass( 'active' )
+				.attr( 'aria-checked', 'true' )
+				.attr( 'tabindex', '0' );
```

Comment:
When the rating group is focused, the focus ring appears only on the selected option, just like in a normal radio group.